src/components/Button.tsx

```diff
@@ -2,7 +2,7 @@ import { Button as AntdButton } from 'antd'
 import { BaseButton } from './styles/Button.styled'
 
 type Props = {
-  text: string
+  text: string | undefined
```

Comment:
is it a bit odd to make text undefined? since the button requires text?

Reply:
da4fb9f80ea397d5a13ca927593b0a76c6e20c21